Framework/DataObjects/src/PeakColumn.cpp

```diff
@@ -206,8 +206,8 @@ void PeakColumn::read(size_t index, const std::string &text) {
 //-------------------------------------------------------------------------------------
 /** @return true if the column is read-only */
 bool PeakColumn::getReadOnly() const {
-  return !((m_name == "h") || (m_name == "k") || (m_name == "l") ||
-           (m_name == "RunNumber"));
+  return ((m_name == "h") || (m_name == "k") || (m_name == "l") ||
```

Comment:
This looks like it returns the opposite bool to before the change, is that intended?